pyproject.toml

```diff
@@ -85,3 +85,7 @@ pep440-version-utils = "^0.3.0"
 semantic_version = "^2.8.5"
 mypy = "^0.991"
 sanic-testing = "^22.3.0, <22.9.0"
+
+
+[tool.pytest.ini_options]
+asyncio_mode = "auto"
```

Comment:
learned something again 😄 thanks for keeping an open eye for interesting simplifications such as this auto asyncio mode.